include/crm/msg_xml.h

```diff
@@ -60,6 +60,7 @@ extern "C" {
  */
 
 #define PCMK_META_ENABLED                   "enabled"
+#define PCMK_META_MIGRATION_THRESHOLD       "migration-threshold"
```

Comment:
It might be and it doesn't matter :) I had thought I remembered your preference going forward being
```
#define CONSTANT value
```

rather than
```
#define CONSTANT<tab><tab>*value
```
for non-numeric values. For example in `msg_xml_compat.h`.

Reply:
Yeah, I just don't care as much about compat headers. They get random stuff added at random times and it's not worth trying to keep them aligned